iam-bootstrap/iam.tf

```diff
@@ -0,0 +1,36 @@
+locals {
+  iam_policy_paths = length(var.iam_policy_paths) == 0 ? ["${path.module}/bootstrap-0.json", "${path.module}/bootstrap-1.json"] : var.iam_policy_paths
+}
+
+data "aws_caller_identity" "admin" {}
+data "aws_partition" "current" {}
+
+resource "aws_iam_policy" "deployment" {
+  count = length(local.iam_policy_paths)
+
+  name = "${var.deploy_id}-deployment-policy-${count.index}"
+
+  policy = templatefile(abspath(pathexpand(local.iam_policy_paths[count.index])), merge({account_id=data.aws_caller_identity.admin.account_id, deploy_id=var.deploy_id, region=var.region, partition=data.aws_partition.current.partition},var.template_config))
+}
+
+resource "aws_iam_role" "deployment" {
+  name = "${var.deploy_id}-deployment-role"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = "sts:AssumeRole"
+        Effect = "Allow"
+        Sid    = ""
+        Principal = {
+          AWS = "arn:aws:iam::${data.aws_caller_identity.admin.account_id}:root"
```

Comment:
govcloud

Reply:
Seems like a miss on the tflint side as it doesn't check this, but should we be running pre-commit in CI to (theoretically) catch this stuff?

Reply:
Fixed this instance, anyhow.